paddle/fluid/inference/analysis/analyzer.h

```diff
@@ -83,6 +83,7 @@ class Analyzer : public OrderedRegistry<PassManager> {
       "conv_bias_mkldnn_fuse_pass",             //
       "conv_relu_mkldnn_fuse_pass",             //
       "conv_elementwise_add_mkldnn_fuse_pass",  //
+      "mkldnn_correct_test_phase_pass",         //
```

Comment:
Should `mkldnn_correct_test_phase_pass` be put before `depthwise_conv_mkldnn_pass` ?

Reply:
`mkldnn_correct_test_phase_pass` can be put anywhere. The later it is called, the less layers there and and thus the less work it has to do.

Reply:
Got it